Files/UserControls/PreviewPane.xaml

```diff
@@ -25,16 +25,12 @@
         </ResourceDictionary>
     </UserControl.Resources>
 
-    <UserControl.Transitions>
-        <TransitionCollection>
-            <PaneThemeTransition x:Name="PreviewPaneTransition" Edge="{x:Bind EdgeTransitionLocation, Mode=OneWay}" />
```

Comment:
If you're removing this, you may as well remove `EdgeTransitionLocation` as well.